SoftLayer/CLI/subnet/create.py

```diff
@@ -10,25 +10,33 @@
 
 
 @click.command(short_help="Add a new subnet to your account")
-@click.argument('network', type=click.Choice(['public', 'private']))
+@click.argument('network', type=click.Choice(['static', 'public', 'private']))
 @click.argument('quantity', type=click.INT)
-@click.argument('vlan-id')
+@click.argument('endpoint-id', type=click.INT)
```

Comment:
Needs a help message about what an endpoint-id is. vlan-id for public/private, subnet-id for static.